gradle.properties

```diff
@@ -245,9 +245,9 @@ slf4jLog4j12Version=1.7.32
 slf4jLog4jApiVersion=1.7.32
 
 
-springBootVersion=2.2.13.RELEASE
+springBootVersion=2.6.1.RELEASE
```

Comment:
@labkey-jeckels Did this work locally? I think they've stopped using the `.RELEASE` suffix on their versions.

Reply:
Also looks like 2.6.1 isn't available for every artifact we use. 2.6.0 might be good for now.

Reply:
It did, but only because I had failed to uncomment the "useEmbeddedTomcat" property. Trying again...